src/Kiota.Builder/Refiners/PythonRefiner.cs

```diff
@@ -140,6 +140,7 @@ public override Task Refine(CodeNamespace generatedCode, CancellationToken cance
     private const string AbstractionsPackageName = "kiota_abstractions";
     private static readonly AdditionalUsingEvaluator[] defaultUsingEvaluators = {
         new (static x => x is CodeClass, "__future__", "annotations"),
+        new (static x => x is CodeClass, "warnings", "warn"),
```

Comment:
maybe conditionally add it when the symbol is deprecated?
you also need to add it for enums I believe